### PR TITLE
New Mathematica Cheat Sheet

### PR DESCRIPTION
**Pull request by pa1pal**
[https://duck.co/ia/view/mathematica_cheat_sheet](IA link)

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pa1pal Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
## [Mathematica Cheat Sheet](https://duck.co/ia/view/mathematica_cheat_sheet)

**Description**: Mathematica is a symbolic mathematical computation program, sometimes called a computer algebra program, used in many scientific, engineering, mathematical, and computing fields.

**Example Query**: [mathematica cheat sheet](https://beta.duckduckgo.com/?q=mathematica cheat sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: http://www.cheat-sheets.org/saved-copy/mathematica-cheat-sheet.pdf

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/mathematica_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/mathematica_cheat_sheet)*

**Comment by moollaza** [comment]
@pa1pal it looks like you've mixed in 2 files here -- please remove the `r-functions.json` file

File: share/goodie/cheat_sheets/json/mathematica.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        "Using Mathematica like a calculator",
+        "Algebra",
+        "Calculus",
+        "Linear Algebra"
+    ],
+
+    "sections": {
+        "Using Mathematica like a calculator": [
+            {
+                "key": "In[2]:= 27.50-11.92",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mathematica.json]: @pa1pal you need to escape all these `[]`

File: share/goodie/cheat_sheets/json/mathematica.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    "name": "Mathematica",
+    "description": "Mathematica is a symbolic mathematical computation program, sometimes called a computer algebra program, used in many scientific, engineering, mathematical, and computing fields.",
+
+    "metadata": {
+        "sourceName": "cheat-sheets.org",
+        "sourceUrl" : "http://www.cheat-sheets.org/saved-copy/mathematica-cheat-sheet.pdf"
+    },
+
+    "aliases": [
+        "mathematica cheat sheets"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mathematica.json]: Please remove this alias

File: share/goodie/cheat_sheets/json/mathematica.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    ],
+
+    "template_type": "code",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mathematica.json]: `terminal` will work better for this lengthy content

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=mathematica cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/mathematica_cheat_sheet_mobile.png)

**Comment by mintsoft** [comment]
@mentions I'm going to close this PR for now; feel free to reopen it whenever you like :+1:

**Pull request closed without merging**